.travis.yml

```diff
@@ -4,8 +4,13 @@ sudo: false
 rvm:
   - 2.1
   - 2.2
-  - 2.3.4
-  - 2.4.1
+  - 2.3
+  - 2.4
```

Comment:
I hope this works but I vaguely remember Travis failing last time I tried this

Reply:
afaik 2.3 is a "static" reference to *some* version 2.3.x.
It will *not* automatically point to the *latest* 2.3.x.

Reply:
It looks like it grabbed 2.3.4 and 2.4.1, which are both one version behind as of September 14th.

Reply:
From https://docs.travis-ci.com/user/languages/ruby/#Specifying-Ruby-versions-and-implementations:

> As we upgrade both RVM and Rubies, aliases like 2.2 or jruby point to different exact versions and patch levels.

So it sounds like it *is* static, but is also periodically updated, so in my opinion it would be ok to leave out the patch version.

Reply:
sgtm